docs/topics/media-pipeline.rst

```diff
@@ -86,17 +86,15 @@ Enabling your Media Pipeline
 .. setting:: IMAGES_STORE
 .. setting:: FILES_STORE
 
-To enable your media pipeline you must first add it to your project
-:setting:`ITEM_PIPELINES` setting.
+To enable your media pipeline you must add its class to the :setting:`ITEM_PIPELINES` setting, like in the following example:
```

Comment:
Unless I’m missing something, it should be more like:

```suggestion
To enable your media pipeline you must add its class import path to the :setting:`ITEM_PIPELINES` setting, like in the following example:
```

And the code changes below must retain the quotes (e.g. `my_project.pipelines.MyImagesPipeline` → `'my_project.pipelines.MyImagesPipeline'`)